static/js/stream_edit.js

```diff
@@ -165,12 +173,6 @@ function show_subscription_settings(sub_row) {
 
     var emails = get_email_of_subscribers(sub.subscribers);
 
-    // If user can not access subscribers no need for search widget.
-    if (!sub.can_add_subscribers) {
-        $("[data-stream-id='" + stream_id + "'] .search").hide();
-    } else {
-        $("[data-stream-id='" + stream_id + "'] .search").show();
-    }
```

Comment:
No need to hide search widget manually, cause we are rendering whole subscription member template, so it will automatically hide widget.